### DISCUSSION TITLE
How to cast nb::dict value to char?

### DISCUSSION THREAD

**Discussion by Gwill** [wjakob/nanobind#179] [Q&A] [answered]
```
#python
input = {}
input[field_name] = 1
```
cast to char, there is some wrong.
![image](https://user-images.githubusercontent.com/692980/231076531-242b6aea-f90e-4cf7-b44c-8fdab928b2fd.png)

cast to int, that is OK
![image](https://user-images.githubusercontent.com/692980/231077178-4b505619-ca63-40e7-9535-e0924fc40e0f.png)

**Comment by wjakob** [comment, marked as answer]
Fixed via https://github.com/wjakob/nanobind/commit/9ae32054d9a6ad17af15994dc51138eb88f71f92. This will be part of the next nanobind release that is probably still a few weeks away. Until then, you can either use the `master` branch of nanobind or implement your own version of `nb::cast` using the example shown in this commit.
